MainModule/Server/Core/Core.lua

```diff
@@ -432,7 +432,7 @@ return function(Vargs, GetEnv)
 				if not p.Parent then
 					return false
 				elseif not parentObj then
-					p:Kick("\n[CLI-102495] Loading Error \nPlayerGui Missing (Waited 10 Minutes)")
+					p:Kick("[CLI-102495] Loading Error - PlayerGui Missing (Waited 10 Minutes)")
```

Comment:
this would look better with a : instead change my mind

Reply:
> this would look better with a : instead change my mind

tried it and it looked ehh